### ISSUE TITLE
Cache doesn't work on non-imageview

### ISSUE THREAD

**Issue by TheFlow95** [hirohisa/ImageLoaderSwift#93]
Hello,

I noticed something weird. When trying to load an image not within an image view, like this:

```
let _ = load(url).completionHandler { url, image, error, cacheType in
    // Do something...
}
```

The cache system doesn't work. But it work using this method :

```
imageView.load(url, placeholder: nil) { url, image, error, cacheType in
    // Do something...
}
```

Is it normal?

**Comment by hirohisa** [comment]
it's normal.

I implement cache work on imageloader request.

I will release 0.12.0.

**Comment by hirohisa** [comment]
I released 0.12.0
